modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,46 @@
+= Audit Logging
+:description: Learn how to use Redpanda's audit logging capabilities.
+
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
+
+Redpanda's auditing capabilities support auditing both administrative and operational interactions with topics and with users. We comply with the open cybersecurity schema framework, providing a predictable and extensible solution which works seamlessly with industry standard tools.
+
+NOTE: All audit logging is configured at the Cluster level. Audit logging is disabled by default.
+
+== Audit Log Flow
+
+The Redpanda audit log mechanism functions similar to the Kafka flow you may be familiar with. When users interact with other users or with topics, either with administrative functions or user functions, Redpanda writes an event to a specialized audit topic. The audit topic is immutable to everything except Redpanda iself -- users are prevented from writing to the topic directly and the Kafka API cannot create or delete it.
+
+image:shared:audit-loging-flow.png[Audit log flow]
+
+By default, any management and authentication actions performed on the cluster are logged to the audit log and are retained for seven days. Interactions with all topics by all principals are audited. Actions performed by the schema registry API, Kafka API, and Admin API are all audited, as are actions performed directly through RPK.
+
+Messages recorded to the audit log topic comply with the https://schema.ocsf.io/[open cybersecurity schema framework]. Any number of analytics frameworks, such as Splunk or Sumo Logic, can receive and process these messages. Using an open standard ensures Redpanda's audit logs coexist with those produced by other IT assets, powering holistic monitoring and analysis of your assets.
+
+== Audit Log Configuration Options
+
+Redpanda's audit logging mechanism supports several options to control the volume of audit logs and the availability of them. Configuration is applied at the cluster level using the standard https://docs.redpanda.com/current/manage/cluster-maintenance/cluster-property-configuration/[cluster configuration mechanism]. Those options exclusive to audit logging include the following:
+
+* `audit_enabled` : Boolean value to enable audit logging (default: false)
+* `audit_log_num_partitions` : Integer value defining the number of partitions used by the audit topic. This configuration is exclusive to the audit log topic. (default: null)
```

Comment:
I would add a note that these only applies to the audit log topic when it is initially created.  Once created, changing this configuration is a noop.